src/ontology/lexmatch/README.md

```diff
@@ -14,30 +14,38 @@
  * Number of mappings in [`unmapped_ncit_mondo_exact`](mondo-only/unmapped_ncit_mondo.tsv): 1177
+ * Number of mappings in [`unmapped_gard_mondo`](mondo-only/unmapped_gard_mondo.tsv): 1
+ * Number of mappings in [`unmapped_gard_mondo_exact`](mondo-only/unmapped_gard_mondo.tsv): 1
 ## mondo_XXXXmatch_ontology
- * Number of mappings in [`mondo_closematch_orphanet`](split-mapping-set/mondo_closematch_orphanet.tsv): 3
- * Number of mappings in [`mondo_broadmatch_orphanet`](split-mapping-set/mondo_broadmatch_orphanet.tsv): 1
- * Number of mappings in [`mondo_exactmatch_orphanet`](split-mapping-set/mondo_exactmatch_orphanet.tsv): 1441
- * Number of mappings in [`mondo_closematch_icd10cm`](split-mapping-set/mondo_closematch_icd10cm.tsv): 361
- * Number of mappings in [`mondo_broadmatch_icd10cm`](split-mapping-set/mondo_broadmatch_icd10cm.tsv): 82
- * Number of mappings in [`mondo_exactmatch_icd10cm`](split-mapping-set/mondo_exactmatch_icd10cm.tsv): 1965
- * Number of mappings in [`mondo_narrowmatch_icd10cm`](split-mapping-set/mondo_narrowmatch_icd10cm.tsv): 58
+ * Number of mappings in [`mondo_closematch_omim`](split-mapping-set/mondo_closematch_omim.tsv): 2
+ * Number of mappings in [`mondo_exactmatch_omim`](split-mapping-set/mondo_exactmatch_omim.tsv): 1286
  * Number of mappings in [`mondo_closematch_ncit`](split-mapping-set/mondo_closematch_ncit.tsv): 72
  * Number of mappings in [`mondo_broadmatch_ncit`](split-mapping-set/mondo_broadmatch_ncit.tsv): 10
  * Number of mappings in [`mondo_exactmatch_ncit`](split-mapping-set/mondo_exactmatch_ncit.tsv): 2310
- * Number of mappings in [`mondo_closematch_omim`](split-mapping-set/mondo_closematch_omim.tsv): 2
- * Number of mappings in [`mondo_exactmatch_omim`](split-mapping-set/mondo_exactmatch_omim.tsv): 1600
  * Number of mappings in [`mondo_closematch_doid`](split-mapping-set/mondo_closematch_doid.tsv): 252
  * Number of mappings in [`mondo_broadmatch_doid`](split-mapping-set/mondo_broadmatch_doid.tsv): 1
  * Number of mappings in [`mondo_exactmatch_doid`](split-mapping-set/mondo_exactmatch_doid.tsv): 163
+ * Number of mappings in [`mondo_closematch_icd10cm`](split-mapping-set/mondo_closematch_icd10cm.tsv): 361
+ * Number of mappings in [`mondo_broadmatch_icd10cm`](split-mapping-set/mondo_broadmatch_icd10cm.tsv): 82
+ * Number of mappings in [`mondo_narrowmatch_icd10cm`](split-mapping-set/mondo_narrowmatch_icd10cm.tsv): 58
+ * Number of mappings in [`mondo_exactmatch_icd10cm`](split-mapping-set/mondo_exactmatch_icd10cm.tsv): 1965
+ * Number of mappings in [`mondo_closematch_gard`](split-mapping-set/mondo_closematch_gard.tsv): 35346
+ * Number of mappings in [`mondo_broadmatch_gard`](split-mapping-set/mondo_broadmatch_gard.tsv): 244
+ * Number of mappings in [`mondo_narrowmatch_gard`](split-mapping-set/mondo_narrowmatch_gard.tsv): 116
+ * Number of mappings in [`mondo_exactmatch_gard`](split-mapping-set/mondo_exactmatch_gard.tsv): 11129
```

Comment:
Do these come from existing `mondo-gard.sssom.tsv` or do these numbers get generated by some logical process in lexmatch?

Reply:
No, it's from `lexmatch` which gets ultimately split.